docs/user/openstack/README.md

```diff
@@ -202,6 +207,26 @@ clouds:
 The file can contain information about several clouds. For instance, the example above describes two clouds: `shiftstack` and `dev-evn`.
 In order to determine which cloud to use, the user can either specify it in the `install-config.yaml` file under `platform.openstack.cloud` or with `OS_CLOUD` environment variable. If both are omitted, then the cloud name defaults to `openstack`.
 
+### Self Signed OpenStack CA certificates
+
+If your OpenStack cluster uses self signed CA certificates for endpoint auth, you will need a few additional steps to run the installer. First, make sure that the host running the installer trusts your CA certificates.
```

Comment:
s/auth/authentication/

Could we link to the OSP docs?

https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/director_installation_and_usage/appe-ssltls_certificate_configuration#Adding_the_Certificate_Authority_to_Clients

Also, I'd like to make it clear this is a limitation -- we shouldn't have to force the node running the installer to add the cert to its system trust -- that we're actively working towards resolving?